pkg/skaffold/deploy/util/namespaces.go

```diff
@@ -69,7 +70,11 @@ func collectHelmReleasesNamespaces(pipelines []latestV1.Pipeline) []string {
 		if cfg.Deploy.HelmDeploy != nil {
 			for _, release := range cfg.Deploy.HelmDeploy.Releases {
 				if release.Namespace != "" {
-					namespaces = append(namespaces, release.Namespace)
```

Comment:
This should use the same approach as in `releaseNamespace()`, and it should surface any errors back up.
https://github.com/GoogleContainerTools/skaffold/blob/main/pkg/skaffold/deploy/helm/util.go#L159-L164